modules/unmute.js

```diff
@@ -0,0 +1,20 @@
+const { GroupSettingChange, MessageType } = require('@adiwajshing/baileys');
+const chalk  = require('chalk');
+
+module.exports = {
+    name: 'unmute',
+    description: 'Unmute',
+    extendedDescription: 'Unmute non-admin members of the group.',
+    async handle(client, chat, BotsApp, args) {
+        if(!BotsApp.isGroup) {
+            client.sendMessage(BotsApp.from, "*.unmute* command is only applicable for a group chat.", MessageType.text)
+            return;
+        }
+        if(!BotsApp.isBotGroupAdmin) {
+            client.sendMessage(BotsApp.from, "Sorry, dont have permissions to do so.", MessageType.text)
```

Comment:
```suggestion
            client.sendMessage(BotsApp.from, "```Sorry, dont have permissions to do so.```", MessageType.text)
```

Reply:
Same as the comment in other file.